gruntfile.js

```diff
@@ -87,9 +89,15 @@ module.exports = function(grunt) {
         },
 
         download: {
+            //One zip for release type (development/master) and one for the version are created here
             openfinZip: {
-                src: ['https://dl.openfin.co/services/download?fileName=StockFlux&config=http://scottlogic.github.io/StockFlux/app.json'],
-                dest: './public/StockFlux.zip'
+                src: ['https://dl.openfin.co/services/download?fileName=StockFlux-' + version +
+                '&config=http://scottlogic.github.io/StockFlux/' + version + '/app.json'],
+                dest: './public/StockFlux-' + version + '.zip'
+            },
+            openfinTypeZip: {
+                src: ['https://dl.openfin.co/services/download?fileName=StockFlux-' + type + '&config=http://scottlogic.github.io/StockFlux/' + type + '/app.json'],
+                dest: './public/StockFlux-' + type + '.zip'
```

Comment:
Does grunt support functions by the way? Looks like with an appropriate input parameter (type/version) this could be less duplicative.